zone/raids.h

```diff
@@ -88,8 +88,8 @@ struct RaidMember{
 	bool IsGroupLeader;
 	bool IsRaidLeader;
 	bool IsLooter;
-	bool IsBot = false;
-	bool IsRaidMainAssistOne = false;
+	bool is_bot = false;
+	bool is_raid_main_assist_one = false;
```

Comment:
Why were only these 2 changed?

Reply:
I'll do the others, was just going through the items Akk had pointed out at first.